go.mod

```diff
@@ -1,10 +1,18 @@
-module github.com/prebid/openrtb/v18
+module github.com/prebid/openrtb/v19
 
-go 1.16
-
-retract v18.0.0 // Published with the wrong module version.
```

Comment:
It seems you may only retract sub versions. A new major version requires to clear all retractions? Doesn't seem right to me, but won't compile otherwise. I can't find documentation on this behavior.